src/NMEA/Info.hpp

```diff
@@ -301,6 +302,13 @@ struct NMEAInfo {
    */
   double humidity;
 
+  //###########
+  //   Engine sensor box
+  //###########
+
+  bool engine_state_available;
```

Comment:
What's the point of this field? It's never initialised, never used, and `EngineState` has its own availability management.

Reply:
Good point, how to proceed? Delete it with a refactor commit?

Reply:
No, amend that commit! (`stg refresh`)

Reply:
Do all commits end up in the master branch? How is the process? Don't they get squashed into one feature commit?

Reply:
No, I don't squash. Having separate commits for different aspects of a PR is very valuable, and that value doesn't go away by merging. Managing the commits and their separation is your task.